TheForceEngine/TFE_A11y/accessibility.cpp

```diff
@@ -146,7 +154,9 @@ namespace TFE_A11Y  // a11y is industry slang for accessibility
 
 		// Try to load the previously selected font.
 		string lastFontPath = TFE_Settings::getA11ySettings()->lastFontPath;
-		tryLoadFont(lastFontPath, false);
+
+		if (lastFontPath, ImGui::GetIO().Fonts->Locked)	{ setPendingFont(lastFontPath);	} 
```

Comment:
that "lastFontPath," is that a copy-paste leftover?

Reply:
Looks like that was supposed to be a `&&` instead of a comma.

Reply:
like this?  (!lastFontPath.empty() && ImGui::GetIO.... ) ?